### DISCUSSION TITLE
suggestion for an addon

### DISCUSSION THREAD

**Discussion by Elitherich** [ScratchAddons/ScratchAddons#4355] [Ideas]
I have an idea for a DEMOTE button in the studios when you don't want to remove a manager but, you want make then a creator this is my idea.

**Comment by WorldLanguages** [comment]
Not possible

**Comment by Elitherich** [comment]
Why?

**Reply by CST1229** [reply]
Because the Scratch API doesn't have the functionality to demote a user. It could kick the user then invite them as curator, but you could just do it manually at that point.

**Reply by WorldLanguages** [reply]
And until they accept the invite, you didn't "demote" them, you removed then invited...

**Comment by Elitherich** [comment]
Ok thank you!

**Reply by DatGamerYoloYT** [reply]
There's a reply feature ;/
